storage/innobase/include/ut0mutex.ic

```diff
@@ -71,6 +77,12 @@ TTASEventMutex<Policy>::wait(
 	and then the event is set to the signaled state. */
 
 	set_waiters();
+	
+	DBUG_EXECUTE_IF("catch_autoinc_mutex_os_lock", {
+			if (m_policy.get_id() == LATCH_ID_AUTOINC) {
+				DEBUG_SYNC_C("autoinc_mutex_wait");
+			}
+		});
```

Comment:
remove the } and align ); to the D in DBUG_EXECUTE_IF

Reply:
done